### PR TITLE
receive a post method of json request.

### PR DESCRIPTION
**Pull request by yangfei4913438**

### CODE REVIEW

File: controller.go [hunk 1]
```diff
@@ -135,6 +137,22 @@ func (c *Controller) RenderTemplate(templatePath string) Result {
 	}
 }
 
+//receive a post method of json request.
```
Comment (xiaozi0lei) [inline controller.go]: // ReceiveJSON unmarshal the input streaming JSON data, returns `map[string]interface{}` type data.
// To access this data we can use a type assertion to access map[string]interface{}. 
// Reference to [Decoding arbitrary data chapter] in https://blog.golang.org/json-and-go

Comment (xiaozi0lei) [inline controller.go]: I think a little clear. :)

**Comment by notzippy** [comment]
Looks like a good start - It would be nice to try and implement a more transparent solution but that can be deferred.

**Comment by brendensoares** [comment]
Covered by #1173

**Pull request closed without merging**